xds/internal/resolver/xds_resolver.go

```diff
@@ -237,7 +237,7 @@ func (r *xdsResolver) run() {
 			return
 		case update := <-r.updateCh:
 			if update.err != nil {
-				r.logger.Warningf("Watch error on resource %v from xds-client %p, %v", r.ldsResourceName, r.client, update.err)
+				r.logger.Warningf("Watch error on resource %v from xds-client %p: %v", r.ldsResourceName, r.client, update.err)
```

Comment:
I think we can skip this. I'm actually working on another PR where I'm getting rid of this log message (along with a few others) as it is redundant.